### ISSUE TITLE
[icetray] do something about I3_REGISTER conflict (Trac #1341)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1333]
Not sure if this is just dead code that needs to be removed.

browser:IceTray/projects/cmake/trunk/tool-patches/common/pybindings.hpp#L219

```text
219	// Hey jvs:  this conflicts with the I3_REGISTER in project icetray
220	// #define I3_REGISTER(r, data, t) BOOST_PP_CAT(register_,t)();
```

Migrated from https://code.icecube.wisc.edu/ticket/1341
```json
{
    "status": "closed", 
    "changetime": "2019-01-12T00:03:00", 
    "description": "Not sure if this is just dead code that needs to be removed.\n\nbrowser:IceTray/projects/cmake/trunk/tool-patches/common/pybindings.hpp#L219\n\n{{{\n219\t// Hey jvs:  this conflicts with the I3_REGISTER in project icetray\n220\t// #define I3_REGISTER(r, data, t) BOOST_PP_CAT(register_,t)();\n}}}", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547251380775367", 
    "component": "cmake", 
    "summary": "[icetray] do something about I3_REGISTER conflict", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-09-11T15:53:19", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150921T12:25:14: 
* jvansanten changed resolution from "" to "fixed"
* jvansanten commented:

> In changeset 2317:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2317"
> Remove dead macros that no one ever ended up using. Long live copy/paste.
> 
> Fixes #1333.
> ```


* jvansanten changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:10: 
* olivas commented:

> Milestone offline V15-12-00 deleted


* olivas changed milestone from "offline V15-12-00" to ""

**Comment by icecube-trac** [comment]
Trac update at 20190112T00:03:00: nwhitehorn commented:

> In changeset 170131:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170131"
> Remove dead macros that no one ever ended up using. Long live copy/paste.
> 
> Fixes #1333.
> ```
